proj/src/main/webapp/static/welcome.html

```diff
@@ -1,75 +1,78 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
-    <title>Welcome</title>
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
+
+    <title>camaraderie</title>
+
+    <!-- Styling -->
     <!-- Bootstrap: Latest compiled and minified CSS -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
-    <!-- Bootstrap: jQuery library -->
+    <!-- Bootstrap: jQuery library --> 
```

Comment:
Doesn't really matter, but could you fix this extra whitespace?

Reply:
ofc